src/brevitas_examples/llm/main.py

```diff
@@ -240,6 +241,11 @@ def quantize_llm(args, extra_args=None):
         require_fx=require_fx and args.export_target is not None,
         device=None)
 
+    # Batched data loader to accelerate GPXQ algorithms
+    if args.gptq or args.gpfq or args.qronos:
+        calibration_batched_loader = get_dataloader_from_dataset(
```

Comment:
Maybe it is worth making it explicit which algorithms leverage this calibration loader at the moment?

```suggestion
        gpxq_calibration_loader = get_dataloader_from_dataset(
```

Reply:
Committed this change. I'm marking the conversation as resolved.